exts/cesium.omniverse/cesium/omniverse/ui/statistics_widget.py

```diff
@@ -53,33 +79,27 @@ def _on_update_frame(self, _e: carb.events.IEvent):
             return
 
         render_statistics = self._cesium_omniverse_interface.get_render_statistics()
-        self._statistics_number_of_materials_loaded_field.set_value(
-            NUMBER_OF_MATERIALS_LOADED_TEXT.format(render_statistics.number_of_materials_loaded)
-        )
-        self._statistics_number_of_geometries_loaded_field.set_value(
-            NUMBER_OF_GEOMETRIES_LOADED_TEXT.format(render_statistics.number_of_geometries_loaded)
-        )
-        self._statistics_number_of_geometries_visible_field.set_value(
-            NUMBER_OF_GEOMETRIES_VISIBLE_TEXT.format(render_statistics.number_of_geometries_visible)
-        )
-        self._statistics_number_of_triangles_loaded_field.set_value(
-            NUMBER_OF_TRIANGLES_LOADED_TEXT.format(render_statistics.number_of_triangles_loaded)
-        )
-        self._statistics_number_of_triangles_visible_field.set_value(
-            NUMBER_OF_TRIANGLES_VISIBLE_TEXT.format(render_statistics.number_of_triangles_visible)
-        )
-        self._statistics_tileset_cached_bytes_field.set_value(
-            TILESET_CACHED_BYTES_TEXT.format(render_statistics.tileset_cached_bytes)
-        )
+        self._materials_loaded_model.set_value(render_statistics.number_of_materials_loaded)
+        self._geometries_loaded_model.set_value(render_statistics.number_of_geometries_loaded)
+        self._geometries_visible_model.set_value(render_statistics.number_of_geometries_visible)
+        self._triangles_loaded_model.set_value(render_statistics.number_of_triangles_loaded)
+        self._triangles_visible_model.set_value(render_statistics.number_of_triangles_visible)
+        self._tileset_cached_bytes_model.set_value(render_statistics.tileset_cached_bytes)
+        self._tileset_cached_bytes_human_readable_model.set_value(render_statistics.tileset_cached_bytes)
+        self._statistics.refresh()
```

Comment:
I'm seeing a drop from 82 FPS to 73 FPS just by having the Cesium Debugging open even after that stray log is removed. This doesn't seem to happen on main. I'm guessing it's related to this line and items being marked as changed every frame.

Reply:
Oh wow, yikes.

I don't think it's related to the refresh. I just commented out all of my usages of `SpaceDelimitedNumberModel` and performance was stable. I actually think it has to do with the fact we're using that f-string stuff. Strings in Python are notoriously slow so it makes sense too. I will rethink my approach for those view models on Monday.